src/APIStatus.js

```diff
@@ -0,0 +1,170 @@
+
+import React,{useState,useEffect} from "react";
+import {MenuItem,IconButton,Typography} from "@material-ui/core";
+import {CheckCircleRounded} from "@material-ui/icons";
+import CircularProgress from '@material-ui/core/CircularProgress';
+import ErrorIcon from "@material-ui/icons/Error";
+import axios from "axios";
+import { apiCorsUrl, APIURL } from "./Devices/hologramConstants";
+import QueryString from "qs";
+import {
+  apiPassword,
+  apiUsername,
+  onfarmAPI,
+} from "./utils/api_secret";
+const hologramAPI = "/api/1/users/me";
+
+const Apistatus = ()=>{
+
+  const [devAPI,setDevAPI] = useState(null);
+  const [gitHubAPI,setGitHubAPI] = useState(null);
+  useEffect(()=>{
+    fetchfromInternal();
+    fetchFromAPIHologram();
+    fetchFromPHPAPI();
+    fetchFromAPIWeather(onfarmAPI);
+    fetchFromAPIGitHub(`https://api.github.com/repos/precision-sustainable-ag/data_corrections/`);
+  },[]);
+
+
+  const [internalAPI,setInternalAPI] = useState(null);
+  const [HologramAPI,setHologramAPI] = useState(null);
+  const [phpAPI,setPHPAPI] = useState(null);
+
+  const fetchfromInternal = () =>{
+              fetch(
+                onfarmAPI + `/raw?table=biomass_in_field&affiliation=MD`
+              ).then((r) => {
+                if (r.headers.get("content-type").split(";")[0] === "text/html") {setInternalAPI(true);
+              }else {setInternalAPI(false);}
+        })
+      }
+
+  const fetchFromAPIHologram = ()=> {
+          axios({
+            method: "post",
+            url: apiCorsUrl + `/watersensors`,
+            data: QueryString.stringify({
+              url: `${APIURL()}${hologramAPI}`,
+            }),
+            headers: {
+              "Content-Type": "application/x-www-form-urlencoded;charset=utf-8",
+            },
+            auth: {
+              username: apiUsername,
+              password: apiPassword,
+            },
+            responseType: "json",
+          })
+            .then((response) => {
+              if (Object.keys(response.data.data).length > 0) {setHologramAPI(true);}
+        else{
+          setHologramAPI(false);
+        }
+        }
+        )
+            }
```

Comment:
the indentation is still not fixed. every function needs to be properly indented and have brackets in the correct places